examples/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "module",
+  "dependencies": {
+    "@concordium/common-sdk": "6.4.0",
+    "@concordium/node-sdk": "6.3.0",
+    "@grpc/grpc-js": "^1.3.4",
+    "@typescript-eslint/eslint-plugin": "^4.28.1",
+    "@typescript-eslint/parser": "^4.28.1",
+    "eslint": "^7.29.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-prettier": "^3.4.0",
+    "lint-staged": "^12.0.2",
+    "meow": "11.0",
+    "prettier": "^2.3.2",
+    "ts-node": "10.9",
+    "typescript": "^4.3.5"
+  },
+  "scripts": {
+    "example": "./execute-example.bash",
+    "lint": "eslint . --cache --ext .ts,.tsx --max-warnings 0",
+    "lint-fix": "yarn lint --fix; exit 0",
+    "build": "tsc",
+    "build-dev": "tsc"
```

Comment:
Yes, but won't we need them to check that they typecheck properly? That is the intention of the lines at least.

Reply:
Yes, but this way we type check the examples every time we build the SDK's, which I think is wrong to do. 
I think it would make sense to do this in a new CI step instead. :thinking:

Reply:
Yes that makes more sense

Reply:
I would call it just `typecheck`or `typecheck:examples` if you need it to be global.